### PR TITLE
Shaving 140KB off boot.js

### PR DESCRIPTION
**Pull request by regiskuckaertz**
It might be a good idea to spread the tip that AB tests that require SVGs should directly call `inlineSVG` rather than import the whole `svgs` module, which is huge. I extracted the function from it, so that it is still usable without the overload.

Before:
picture 2

After:
picture 1

@sndrs

### CODE REVIEW

**Comment by sndrs** [comment]
💥 👀

File: static/src/javascripts/projects/common/views/svg.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+define(function () {
+    return svg;
+
+    function svg(markup, classes, title) {
+        // Only mess with classes if we actually need to.
+        if (classes) {
+            if (Array.isArray(classes)) {
```
Comment (piuccio) [inline static/src/javascripts/projects/common/views/svg.js]: I've probably asked this many times. Do we include a polyfill for `Array.isArray` in IE8? can't remember

**Comment by piuccio** [comment]
looks really good. The only minor comment is that you've mixed some unrelated changes (remove `$`, `reduce`).

If they're not risky or hard to test, go for it! 👍

File: static/src/javascripts/projects/common/views/svg.js (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/views/svg.js]: This is part of `es5-shim` yes

**Comment by regiskuckaertz** [comment]
It's kind of related, I should have mentioned that the SVG thing is just one aspect of this PR 😅 

Thanks bro!

**Pull request merged**